sections/elements.include

```diff
@@ -1244,6 +1257,20 @@
     </tr>
 
     <tr>
+     <th><a element lt="picture source"><code>source</code></a></th>
+     <td>Image source for <{img}></td>
+     <td>none</td>
+     <td><{picture}></td>
+     <td>empty</td>
+     <td><a lt="global attributes">globals</a>;
+         <{source/srcset}>;
+         <{source/sizes}>;
+         <{source/media}>;
+         <a element-attr lt='picture source type'><code>type</code></a></td>
+     <td><a href='#Picture-HTMLSourceElement'><code class='idl'>HTMLSourceElement</code></a></td>
```

Comment:
This is the problem on where there is a partial. I think we should link this as {{HTMLPictureElement/HTMLSourceElement}}